docs/schema.md

```diff
@@ -765,89 +765,9 @@ contains a JSON object with unspecified fields.
-    "references": [
-        {"type": "REPORT", "url": "https://golang.org/issue/43786"},
-        {"type": "WEB", "url": "https://github.com/catenacyber/elliptic-curve-differential-fuzzer"}
-    ],
-    "affected": [ {
-        "package": {
-            "ecosystem": "Go",
-            "name": "crypto/elliptic"
-        },
-        "ranges": [
-            {
-                "type": "SEMVER",
-                "events": [
-                    { "introduced": "1.0.0" },
-                    { "fixed": "1.14.14" },
-                    { "introduced": "1.15.0" },
-                    { "fixed": "1.15.17" }
-                ]
-            }
-        ],
-        "ecosystem_specific": {
-            "functions": ["P224"],
-            "module": "std",
-            "severity": "HIGH"
-        }
-    } ]
-}
-```
-
-## Go tool vulnerability
-
-The shared format can also be used to describe vulnerabilities in commands and
-applications.  Here is an entry for a recent Go tool vulnerability:
-
-```json
-{
-    "schema_version": "1.2.0",
-    "id": "GO-2021-99999",
-    "published": "2021-01-21T19:15:00Z",
-    "modified": "2021-03-10T23:20:53Z",
-    "aliases": ["CVE-2021-3115"],
-    "summary": "packages using cgo can cause arbitrary code execution at build time",
-    "details": "The go command may execute arbitrary code at build time when cgo is in use on Windows. This may occur when running \"go get\", or any other command that builds code. Only users who build untrusted code (and don’t execute it) are affected.\n\nIn addition to Windows users, this can also affect Unix users who have \".\" listed explicitly in their PATH and are running \"go get\" or build commands outside of a module or with module mode disabled.\n\nThanks to RyotaK (https://twitter.com/ryotkak) for reporting this issue.",
-    "references": [
-        {"type": "REPORT", "url": "https://golang.org/issue/43783"}
-    ],
-    "affected": [ {
-        "package": {
-            "ecosystem": "Go",
-            "name": "cmd/go"
-        },
-        "ranges": [
-            {
-                "type": "SEMVER",
-                "events": [
-                    { "introduced": "1.0.0" },
-                    { "fixed": "1.14.14" },
-                    { "introduced": "1.15.10" },
-                    { "fixed": "1.15.17" }
-                ]
-            }
-        ],
-        "ecosystem_specific": {
-            "severity": "HIGH"
-        }
-    } ]
-}
-```
+The [Go vulnerability database](https://go.dev/security/vulndb/) uses this format.
```

Comment:
Just a heads up that we changed the canonical URLs for:

https://go.dev/security/vulndb -> https://go.dev/security/vuln.
https://go.dev/security/vulndb/api -> https://go.dev/security/vuln/database

The old links will still redirect.

Reply:
Thanks, fixed!